pihole_adlist_tool

```diff
@@ -833,26 +837,26 @@ if [ "$FURTHER_ACTION" -eq 3 ]; then
     # remove all domains from gravity_dup that are also contained in that adlist
     # count how many domains are still on gravity_dup
 
-    adlist_conf_minimal_enabled=(`sqlite $TEMP_DB "select id from adlist where unique_domains_covered IS NOT NULL;"`)
+    mapfile -t adlist_conf_minimal_enabled < <(sqlite $TEMP_DB "select id from adlist where unique_domains_covered IS NOT NULL;")
 
     sqlite $TEMP_DB "CREATE TABLE gravity_dup AS SELECT * FROM gravity_strip"
 
     for adlist_id in "${adlist_conf_minimal_enabled[@]}"; do
        sqlite $TEMP_DB "DELETE FROM gravity_dup WHERE domain IN (SELECT domain from gravity_dup where adlist_id=$adlist_id);"
     done
 
-    LEFT_DOMAINS=(`sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;"`)
+    mapfile -t LEFT_DOMAINS < <(sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;")
```

Comment:
I think `LEFT_DOMAINS` is not an array, it's a single number

Reply:
My local `shellcheck` complained:
```
In pihole_adlist_tool line 849:
    LEFT_DOMAINS=(`sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;"`)
                  ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
                  ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean:
    LEFT_DOMAINS=($(sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;"))

  https://www.shellcheck.net/wiki/SC2207 -- Prefer mapfile or read -a to spli...
  https://www.shellcheck.net/wiki/SC2006 -- Use $(...) notation instead of le...
```

After fixing SC2006:
```
In pihole_adlist_tool line 849:
    LEFT_DOMAINS=($(sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;"))
                  ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).

For more information:
  https://www.shellcheck.net/wiki/SC2207 -- Prefer mapfile or read -a to spli...
```

Reply:
@yubiuser , could you please check your comment with my `shellcheck`?

Reply:
@yubiuser , had you time to review this?

Reply:
Do the following: `declare -i LEFT_DOMAINS` right at the beginning, then `LEFT_DOMAINS=$(sqlite $TEMP_DB "SELECT COUNT (domain) from gravity_dup;")` and `while [[ $LEFT_DOMAINS != 0 ]]; do`

Reply:
done in https://github.com/yubiuser/pihole_adlist_tool/pull/37/commits/54a626c18e278bd9baf549cb1e8cfb54ae9558a1

Reply:
done in https://github.com/yubiuser/pihole_adlist_tool/pull/37/commits/54a626c18e278bd9baf549cb1e8cfb54ae9558a1